pkg/kademlia/routing.go

```diff
@@ -202,3 +205,19 @@ func (rt *RoutingTable) GetBucketTimestamp(bIDBytes []byte) (time.Time, error) {
 func (rt *RoutingTable) iterate(opts storage.IterateOptions, f func(it storage.Iterator) error) error {
 	return rt.nodeBucketDB.Iterate(opts, f)
 }
+
+// ConnFailure implements the Transport failure function
+func (rt RoutingTable) ConnFailure(ctx context.Context, node *pb.Node, err error) {
```

Comment:
Let's keep the receivers consistent and change (rt RoutingTable) to (rt *RoutingTable)

Reply:
Same point as above.